tests/integration/test_password_rotation.py

```diff
@@ -39,13 +43,17 @@ async def test_deploy_active(ops_test: OpsTest):
         await ops_test.model.wait_for_idle(apps=[APP_NAME], status="active", timeout=1500)
 
 
+@pytest.mark.runner(["self-hosted", "linux", "X64", "jammy", "large"])
```

Comment:
@dragomirp Q: is it necessary here? AFAIK, it is set for group 1 in line 28 above.

Reply:
No, one self hosted mark per suite should be sufficient.